src/qibolab/serialize.py

```diff
@@ -48,10 +51,12 @@ def load_qubits(
         q: Qubit(q, **char)
         for q, char in runcard["characterization"]["single_qubit"].items()
     }
+
     if extras_folder is not None:
-        single_qubit = runcard["characterization"]["single_qubit"]
-        for qubit in qubits.values():
-            qubit.kernel_path = extras_folder / single_qubit[qubit.name]["kernel_path"]
+        kernels = Kernels.load(path=extras_folder / KERNELS_FILE)
```

Comment:
I guess it's really a matter of taste.

But you can do the things you asked even with the current strategy. Just have multiple kernel files inside the `extras_folder`, and rename those to `kernels.npz`.

About the idea of replacing a folder with a file, since there is nothing else, in general I'd agree with you. But here it would be already better to have a single file per runcard (to avoid spreading them), and we start from two (the `.py` and `.yml`).

Reply:
Maybe I have a counterproposal: what if we keep the directory, but we move all the files in there?

I.e.
```sh
iqm5q/
  platforms.py
  parameters.yml
  kernels.npz
```

At least it would be easy to collect all the related files, and in case you want to preserve/circulate a platform, you could always `tar`/`zip` the folder and grab it as a single file.

Reply:
It seemed more future proof as we may need more stuff to be in that folder, and also what @alecandido mentioned would be more organized so we can do it on another PR on the qrc_cluster platforms.

Reply:
> ```shell
> iqm5q/
>   platform.py
>   parameters.yml
>   kernels.npz
> ```

In my opinion, this is the best solution, but I agree doing it in another PR because it requires a few more changes which are unrelated to the kernels.

Reply:
@alecandido @stavros11 @Jacfomg  if everyone agrees I can open a PR in qibocal to generate directly folders and not just platform runcards. I believe that it should be feasible. It should be useful for the qibocal user to be able to see directly the create_function alongside the parameters and the kernels.

Reply:
Indeed, it would be useful. I would just wait to do it after we implement the structure proposed above (one directory per platform) in qibolab and in the platforms. Also, I think it makes more sense to implement the function you are proposing in qibolab, perhaps replacing the current `dump_runcard` and just use it qibocal, because at the end of the day it solves the problem of platform serialization which is a qibolab problem. However, this is a bit orthogonal to #762, so let’s wait to see what others think.

Reply:
You are right. The function should be in qibolab. I will wait.

Reply:
I agree with you that we should try to roll it out, and we should try to do it in Qibolab.
Btw, if it's going through Qibolab it should be also a non-breaking change, since Qibocal should not care much about the runcard serialization structure.

> However, this is a bit orthogonal to #762, so let’s wait to see what others think.

I would say orthogonal is the correct word, not opposite. The moment we implement the folder, we could well dump the parameters in JSON, and drop the PyYAML dependency as well.